ubuntu/ubuntu-ruby-2.7/Dockerfile

```
@@ -0,0 +1,58 @@
+FROM ubuntu:xenial
+
+LABEL MAINTAINER takemi.ohama@ietty.co.jp
+
+ENV TERM xterm
+ENV DEBIAN_FRONTEND noninteractive
```

[nits]
ENV は `=` をつかった記法が推奨されてるようです。

https://docs.docker.jp/engine/reference/builder.html#env
```suggestion
ENV TERM=xterm
    DEBIAN_FRONTEND=noninteractive
```